care/emr/api/viewsets/patient_identifier.py

```diff
@@ -56,7 +56,9 @@ def authorize_create(self, instance):
 
     def authorize_update(self, request_obj, model_instance):
         if model_instance.facility:
-            facility = get_object_or_404(Facility, external_id=model_instance.facility)
+            facility = get_object_or_404(
```

Comment:
We already have the object, why are we fetching it again ? I think it was wrong from the get go.